src/Table/Table.component.tsx

```diff
@@ -276,6 +282,23 @@ const SWrapperDiv = styled(ReactTable)`
   &.ReactTable .rt-noData {
     display: none !important;
   }
+
+  ${props =>
+    props.stickyHeaders &&
+    css`
+      &.ReactTable .rt-table {
+        overflow: visible;
+        border-top: none;
+      }
+
+      &.ReactTable .rt-thead.-header {
+        position: sticky;
+        top: 0;
+        z-index: 1;
+        background-color: white;
```

Comment:
I don't think `background-color: white;` is needed - it's already defined on the regular `&.ReactTable .rt-thead.-header`

Reply:
It looks like we will need 
        background-color: white; here. I tried removing it and now you can see everything behind the header as you scroll. That's what prompted me to set it there.

Reply:
![image](https://user-images.githubusercontent.com/7967701/228354011-c6f2416b-d0ce-4202-9ac8-31a3d5793723.png)

Reply:
Ah ok -- the color value should be coming from `    background-color: ${(props: any) =>
      props.theme.table.header.backgroundColor};` instead of being hardcoded

Reply:
Ah good point! I'll update that.

Reply:
Hmm, I don't see that prop defined in any of the theme JSON files.

Here's what we currently have in table.headers:

```
table: {
    header: {
      borderBottomColor: colors.drk800,
      color: colors.drk800,
      fontWeight: 'bold',
      fontSize: '16px',
    }
}
```

Should I add a `backgroundColor` prop to the base theme?

Reply:
I added the backgroundColor prop to the themes and it worked.